fixing docs

sofia-calgaro opened the pull request.

gipert commented on docs/source/conf.py:
```diff
@@ -5,8 +5,7 @@
 
-# sys.path.insert(0, Path(__file__).parents[2].resolve().as_posix())
-sys.path.insert(0, Path(__file__).resolve().as_posix())
+sys.path.insert(0, Path("../../src").resolve().as_posix())
```
Questo dovrebbe funzionare com'era prima...

gipert commented on setup.cfg:
```diff
@@ -31,6 +31,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+    fcutils@git+https://github.com/legend-exp/pyfcutils@v0.1.0#egg=fcutils
```
`fcutils` non viene trovato perchè è una dipendenza opzionale di pygama. Dovreste installare `pygama[daq]` invece che solo `pygama`. Tipo (non testato):
```
pygama[daq]@git+https://github.com/legend-exp/pygama@v1.0.0#egg=pygama
```

gipert commented on src/legend_data_monitor/__main__.py:
```diff
@@ -1,6 +1,6 @@
 """legend_data_monitor.__main__: executed when legend_data_monitor directory is called as script."""
 
 
-from .legend_data_monitor import main
+from . import main
 
 main()
```
Per creare eseguibili vi consiglio di dichiararli in `setup.cfg`. C'è già qualcosa impostato come `console_script` ma non mi sembra funzionante...

sofia-calgaro commented:
- Updated the code to skip events when no files/channels/parameters are present in the file
- Added messages on terminal
- Added the study of the ch000 only
- Fixed the selection of pulser/physical data
- Documentation will be adjusted after having added the executable in the "proper" manner
